src/components/ModalConnectButton/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { InjectedConnector, useStarknet } from "@starknet-react/core";
```

Comment:
Correct me if I'm wrong but I guess this component is the content of the registration modal where the user is prompted to choose between the two wallet connection.
So this should be named after this intent, something like `MultiWalletConnection`?

Second thing about this folder, since it's only used by the `RegisterModal`, it should be a child folder of the `RegisterModal` folder – [fractal components rule](https://github.com/onlydustxyz/marketplace-frontend/pull/31)

Reply:
Ok! Will do these changes and commit, thank you for the feedback